apps/firefox/firefox.win.talon

```diff
@@ -27,7 +27,12 @@ action(browser.bookmarks_bar):
 
 action(browser.focus_address): 
 	key(ctrl-l)
-	
+
+tab search <user.text>:
+	browser.focus_address()
+	"% {text}"
```

Comment:
this is pretty nitpicky, but for multiline commands I prefer to use explicit `insert` calls